### PR TITLE
Create ts4_40k.json

### PR DESCRIPTION
**Pull request by mazinskihenry**
Need to make another json file for the 40k MCC and future operations

### CODE REVIEW

File: traycer/arma3/training/ts4_40k.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "modpack_name":"ts1_40k",
```
Comment (ColdEvul) [inline traycer/arma3/training/ts4_40k.json:2]:
```suggestion
    "modpack_name":"ts4_40k",
```
@Jarvvski Correct?

Comment (uaon) [reply traycer/arma3/training/ts4_40k.json:2]: that only applies to the preset generator, which is a script we don't really utilize on the image for the container(s)

Comment (uaon) [reply traycer/arma3/training/ts4_40k.json:2]: You can put whatever in there

**Review by uaon — CHANGES_REQUESTED** [review]
Id reduce the size of the modnames, just because I'm unsure what the string/char limit is for steam.

Example:
``@imperialconquest --> @impc``
``@eventhandler --> @eh``

**Review by uaon — APPROVED** [review]

**Review by RaduM84 — APPROVED** [review]
Agreed on the ts1 to ts4 change requested above, if only for consistency.

**Review by RaduM84 — APPROVED** [review]

File: traycer/arma3/training/ts4_40k.json (hunk 1 above)
Comment (ColdEvul) [reply traycer/arma3/training/ts4_40k.json:2]: utalizing same as name then

**Pull request merged**